src/components/search.js

```diff
@@ -47,6 +79,11 @@ export default class Search extends React.PureComponent {
           placeholder={i18n.search}
           autoFocus={autoFocus}
         />
+        <div
+          className="emoji-mart-search-icon"
+          onClick={this.handleClick}
+          dangerouslySetInnerHTML={{ __html: icon }}
+        />
```

Comment:
Can we make this element "tabbable" (being able to select it with "tab"). 
The behavior I'd like is actually already in Storybook, in their "filter" box. 

The "magnifying glass" should probably not be "tabbable" though.

![make-search-accessible](https://user-images.githubusercontent.com/8042228/41994914-fd1c7300-7a1d-11e8-8854-06e137cab31c.gif)